packages/celotool/geth_tests/transfer_tests.ts

```diff
@@ -523,13 +511,13 @@ describe('transfer tests', function(this: any) {
         })
 
         describe('when paying for gas in Celo Dollars', () => {
-          const intrinsicGas = 156000
+          const intrinsicGas = 291000
```

Comment:
Honestly a little surprised myself. There should be less storage read/writes since these numbers use single storage slots. 

I don't have a great intuition for how expensive calls are. There *are* a lot more calls happening now because the library is linked.